src/views/Backups/index.ts

```diff
@@ -0,0 +1,5 @@
+export { default } from './BackupsView';
+export { default as BackupsAsFoldersList } from './components/BackupsAsFoldersList';
+export { default as BackupDropdownActions } from './components/BackupDropdownActions';
+export { default as DeviceDropdownActions } from './components/DeviceDropdownActions';
+export { default as DeviceList } from './components/DeviceList';
```

Comment:
Do the same as [here](https://github.com/internxt/drive-web/pull/1698#discussion_r2459119805), separate the barrel